Quaver.Shared/Screens/Edit/UI/Panels/EditorPanelColorPicker.cs

```diff
@@ -0,0 +1,228 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Quaver.API.Enums;
+using Quaver.API.Maps.Structures;
+using Quaver.Shared.Assets;
+using Quaver.Shared.Graphics.Notifications;
+using Quaver.Shared.Helpers;
+using Quaver.Shared.Screens.Edit.Actions;
+using Quaver.Shared.Screens.Edit.Actions.Colors.Add;
+using Quaver.Shared.Screens.Edit.Actions.Hitsounds.Add;
+using Quaver.Shared.Screens.Edit.Actions.Hitsounds.Remove;
+using Wobble.Bindables;
+using Wobble.Graphics;
+using Wobble.Graphics.Sprites;
+using Wobble.Graphics.Sprites.Text;
+using Wobble.Graphics.UI.Buttons;
+using Wobble.Managers;
+using DrawingColor = System.Drawing.Color;
+
+namespace Quaver.Shared.Screens.Edit.UI.Panels
+{
+    public class EditorPanelColorPicker : EditorPanel
+    {
+        /// <summary>
+        /// </summary>
+        private BindableList<HitObjectInfo> SelectedHitObjects { get; }
+
+        /// <summary>
+        /// </summary>
+        private EditorActionManager ActionManager { get; }
+
+        /// <summary>
+        /// </summary>
+        private List<DrawableEditorColorPicker> HitsoundList { get; }
+
+        /// <summary>
+        /// </summary>
+        private Bindable<HitSounds> SelectedHitsounds { get; } = new Bindable<HitSounds>(0);
+
+        /// <inheritdoc />
+        /// <summary>
+        /// </summary>
+        /// <param name="selectedHitObjects"></param>
+        /// <param name="manager"></param>
+        public EditorPanelColorPicker(BindableList<HitObjectInfo> selectedHitObjects, EditorActionManager manager)
+            : base("Snap Colors")
+        {
+            ActionManager = manager;
+            SelectedHitObjects = selectedHitObjects;
+
+            Depth = 1;
+
+            HitsoundList = new List<DrawableEditorColorPicker>
+            {
+                new DrawableEditorColorPicker(SnapColor.Red, SelectedHitObjects, ActionManager),
```

Comment:
Can't build the program. It seems `SnapColor` was left undefined. Missing commit?

Reply:
I just hacked together the proper enum to get it to build for now